libsoftwarecontainer/src/gateway/network/networkgatewayparser.cpp

```diff
@@ -73,56 +63,38 @@ ReturnCode NetworkGatewayParser::parseNetworkGatewayConfiguration(const json_t *
         }
     }
 
-    std::string readTarget;
-    if (!JSONParser::read(element, "default", readTarget)) {
-        log_error() << "No default target specified or default target is not a string.";
-        return ReturnCode::FAILURE;
-    }
-
-    e.m_defaultTarget = parseTarget(readTarget);
-    if (e.m_defaultTarget == IPTableEntry::Target::INVALID_TARGET
-        || e.m_defaultTarget == IPTableEntry::Target::REJECT) {
-        log_error() << "Default target '" << readTarget << "' is not a supported target";
-        return ReturnCode::FAILURE;
-    }
-
     return ReturnCode::SUCCESS;
 }
 
 
 ReturnCode NetworkGatewayParser::parseRule(const json_t *element, std::vector<IPTableEntry::Rule> &rules)
 {
     IPTableEntry::Rule r;
-    std::string target;
-    if (!JSONParser::read(element, "target", target)) {
-        log_error() << "Target not specified in the network config";
-        return ReturnCode::FAILURE;
-    }
 
-    r.target = parseTarget(target);
-    if (r.target == IPTableEntry::Target::INVALID_TARGET) {
-        log_error() << target << " is not a valid target.";
-        return ReturnCode::FAILURE;
-    }
+    r.target = IPTableEntry::Target::ACCEPT;
 
     if (!JSONParser::read(element, "host", r.host)) {
         log_error() << "Host not specified in the network config.";
         return ReturnCode::FAILURE;
     }
 
-    // Parsing different port formats
-    json_t *port = json_object_get(element, "port");
+    // Parsing port formats
+    json_t *port = json_object_get(element, "ports");
```

Comment:
Is this a list of ports or a single port? The use of both "port" and "ports" is confusing to me.